.github/workflows/libunifex-ci.yml

```diff
@@ -82,6 +82,22 @@ jobs:
             cc: "gcc-10", cxx: "g++-10",
             cmake_args: "-D CMAKE_CXX_STANDARD:STRING=20 -D CMAKE_CXX_FLAGS:STRING=-fcoroutines"
           }
+        - {
+            name: "Linux GCC 10 Debug (C++20) w/coroutines w/liburing", artifact: "Linux.tar.xz",
```

Comment:
I do wonder how many various combinations here we realistically need to support for CI.  I'd be fine if we condense this down into a single `w/coroutines` mode.

Might as well get this commit in as-is though to resolve the compile issues.